src/program.ts

```diff
@@ -3639,6 +3639,10 @@ export class Local extends VariableLikeElement {
     assert(type != Type.void);
     this.setType(type);
   }
+
+  isClosure(flow: Flow): bool {
```

Comment:
I'm not sure that's a good name for this routine. May be "isFree" (free variable) or "isOut" will be better?